src/components/PopupWithForm.js

```diff
@@ -28,4 +30,8 @@ export default class PopupWithForm extends Popup {
     });
```

Comment:
Please, delete `this._popUpForm.reset();` from line 29 in `PopupWithForm.js`

You need to **reset** the form only in block `then` not to do that immediately after `submit` because when there is an error the popup will still be opened and the user can try to send the data again to the server without having to type the data into the inputs once more